.ci/docker/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -exu
+
+IMAGE_NAME="$1"
+shift
+
+export CUDA_VERSION="12.1.1"
+export UBUNTU_VERSION="20.04"
+
+export BASE_IMAGE="nvidia/cuda:${CUDA_VERSION}-devel-ubuntu${UBUNTU_VERSION}"
```

Comment:
NVIDIA image contains tons of dev tools which are not really needed to build/test tutorials, are they?
```suggestion
export BASE_IMAGE="ubuntu:${UBUNTU_VERSION}"
```